misk-hibernate-testing/src/main/kotlin/misk/hibernate/StartVitessService.kt

```diff
@@ -82,7 +82,7 @@ internal class DockerVitessCluster(
       return
     }
 
-    docker.killContainerCmd(containerId!!).exec()
+    docker.removeContainerCmd(containerId!!).exec()
```

Comment:
With docker command line if you try to remove a running container it'll fail, but I'll assume this works.